core/new-gui/src/app/dashboard/service/user-file/user-file.service.ts

```diff
@@ -19,7 +19,7 @@ export const USER_FILE_NAME_UPDATE_URL = `${USER_FILE_BASE_URL}/update/name`;
   providedIn: "root",
 })
 export class UserFileService {
-  private dashboardUserFileEntryChanged = new Subject<void>();
+  private dashboardUserFileEntryChanged = new BehaviorSubject<void>(undefined);
```

Comment:
can we use a ReplaySubject instead?

Reply:
Behavior subject is better because it will always try to fetch it once initially. Replay subject can only do it if there has been a change.